assets/bigquery/fishing-events-2-merge-schema.json

```diff
@@ -0,0 +1,72 @@
+[
+    {
+        "name": "ssvid",
+        "type": "STRING",
+        "description": "Source specific vessel id. This is the transponder id, and for AIS this is the MMSI"
+    },
+    {
+        "name": "seg_id",
+        "type": "STRING",
+        "description": "Id of the segment involved in the event."
+    },
+    {
+        "name": "timestamp",
```

Comment:
Technically, at this point the table only contains messages (with event start and end timestamp columns). So this timestamp (and year, meters_to_prev and hours) all refer to messages and not events.